openstack/vpn/v5/connection_health_checks/requests.go

```diff
@@ -0,0 +1,10 @@
+package connection_health_checks
```

Comment:
just use the API name in sdk, the sdk should only translate the API and do nothing more
```suggestion
package connection_monitors
```

Reply:
down